python/app/dialog.py

```diff
@@ -1059,6 +1064,16 @@ def build_entity_tabs(self):
                     filter_menu = ShotgunFilterMenu(
                         data.get("view"), bg_task_manager=self._task_manager
                     )
+                    named_filters = self._app.execute_hook_method(
+                        "shotgun_filters_hook",
+                        "get_named_filters",
+                        tab_name=entity_tab_name,
+                    )
+                    if type(named_filters) is dict:
+                        filter_menu.set_named_filters(named_filters)
+
+                    # TODO: this will mean the app requires the updated QTwidgets framework
```

Comment:
So please update the info.yml accordingly. Plus document the hook in the file too.

Reply:
It's been a while since I've done this but I assume I can only do this once I have a QtWidgets release version to target? This PR will need to wait until the other is released I think?